### ISSUE TITLE
Upstream: a11y(critical) authoring card toggle has no accessible name

### ISSUE THREAD

**Issue by bmtcril** [openedx/end-to-end-tests#55]
https://github.com/openedx/frontend-app-authoring/issues/3240
